### ISSUE TITLE
NFM 模型的 modules.py 中未导入 Layer

### ISSUE THREAD

**Issue by cqx15390819800** [ZiyaoGeng/RecLearn#44]
from tensorflow.keras.layers import Dense, Dropout
改为
from tensorflow.keras.layers import Dense, Dropout, Layer

**Comment by ZiyaoGeng** [comment]
感谢指出
